content/commands/hpexpiretime/index.md

```diff
@@ -47,7 +47,7 @@ key_specs:
       limit: 0
     type: range
 linkTitle: HPEXPIRETIME
-since: 8.0.0
+since: 7.4 RC1
 summary: Returns the expiration time of a hash field as a Unix timestamp, in msec.
```

Comment:
For consistency with other commands - please use either (seconds) / (milliseconds) or ", in seconds" / ", in milliseconds"

Reply:
Done. Next push.